plugins/catalog-backend/src/database/DefaultProcessingDatabase.ts

```diff
@@ -145,20 +145,22 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
       BATCH_SIZE,
     );
 
-    // Delete old refresh keys
-    await tx<DbRefreshKeysRow>('refresh_keys')
-      .where({ entity_id: id })
-      .delete();
+    if (refreshKeys && refreshKeys.length > 0) {
```

Comment:
Should this just be `if (refreshKeys) {` perhaps? With this I take it there's no way to intentionally clear out existing refresh keys. Actually, how come this change is needed? The actual fix is in the processor right?

Reply:
Hm, yea, that's correct, I was just thinking to add this to try to prevent a scenario where if you do not return refresh keys we won't delete the existing ones. 
But you are correct with this we won't be able to intentionally delete them, that's not good, let me remove this line!